examples/compiled/area_density_stacked.vg.json

```diff
@@ -44,7 +44,7 @@
       "type": "group",
       "from": {
         "facet": {
-          "name": "faceted_path_main",
+          "name": "faceted_path_Main",
```

Comment:
Better keep the lowercase

Reply:
Yup, I am realizing that is the cause of most of my test failures. Will keep lowercase.